### PR TITLE
Initial Javascript View Development

### PR DESCRIPTION
**Pull request by ohler55**

### CODE REVIEW

**Comment by ashmaroli** [comment]
I see that you squashed all the commits..

**Comment by ohler55** [comment]
yes, cleaner that way.

**Comment by ashmaroli** [comment]
true.. but I would've squashed only small related changes.. now this commit diff is one huge set of changes.. oh well..

File: view/pages/design_notes.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- [x] create `view/index.html`
+- [ ] `wab.View`
+- [ ] `wab.Page`
+- [ ] `wab.Obj`
+- [ ] show `const` string first
+- [ ] `wab.Sender`
+- [ ] `wab.List` request and display raw JSON list
+- [ ] `wab.List` better display
+- [ ] `wab.Obj` handle template and display
```
Comment (ashmaroli) [inline view/pages/design_notes.md]: is this section required to be merged?

Comment (ohler55) [inline view/pages/design_notes.md]: Probably not.

File: view/pages/proto-list.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
+<!-- Template for list-table -->
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Welcome to WABuR</title>
+  <link rel="stylesheet" type="text/css" media="screen, print" href="wab.css" />
+  <link rel="stylesheet" type="text/css" media="screen, print" href="assets/wabfont/style.css" />
+</head>
+
```
Comment (ashmaroli) [inline view/pages/proto-list.html]: This entire file can be removed..

**Comment by ohler55** [comment]
Now just waiting for travis to finish.

**Comment by ohler55** [comment]
Over 3 hours waiting for travis. Enough.

**Pull request merged**